src/Stormpath/Resource/ApiKey.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * Copyright 2013 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Stormpath\Resource;
+
+
+use Stormpath\Authc\Api\ApiKeyEncryptionUtils;
+use Stormpath\Stormpath;
+
+class ApiKey extends InstanceResource implements Deletable
```

Comment:
In favor of this api key resource, we should be deprecating the one on the Stormpath folder and document that it will go away after the first final version (when we remove "beta"). Something similar to what we did on the Java SDK. This is because these two classes represent api keys, only that one is a Resource instance and the other one is not. 

That means that, after removing the deprecated class, this class will also be the one used for authenticating the SDK calls to Stormpath.